src/Modules/controllTools.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] **Required:** You should not disable the linters, they provide a standard for team/community code here at Microverse and in your future workplaces. Kindly fix the errors the point out.

Reply:
Hey @brytebee for this part I found the solution only for two but for `eslint-disable import/no-cycle` I can't remove it now because when I remove it I found the error when I run build and if I try to fix the error of this one after fixing it I found the errors of linters, I try to fix all both but didn't found the solution. So I don't know if you have any suggestions. I removed only lines 2 and 3.